ansible_ai_connect_chatbot/index.html

```diff
@@ -22,6 +22,6 @@
       <script type="module" src="/src/index.tsx"></script>
     </div>
     <div id="user_name" hidden>{{user_name}}</div>
-    <div id="bot_name" hidden>{{bot_name}}</div>
+    <div id="bot_name" hidden>Ansible Lightspeed</div>
```

Comment:
Please keep the original line (`{{bot_name}}`) here.  It will be replaced with the value of envvar `ANSIBLE_AI_CHATBOT_NAME` with the change made with #1372

Reply:
updated